src/protocols/secure_channel/CASEServer.cpp

```diff
@@ -181,4 +179,34 @@ void CASEServer::OnSessionEstablished(const SessionHandle & session)
                     ChipLogValueScopedNodeId(session->GetPeer()));
     PrepareForSessionEstablishment(session->GetPeer());
 }
+
+void CASEServer::SendBusyStatusReport(Messaging::ExchangeContext * ec)
+{
+    using namespace Protocols::SecureChannel;
+
+    // 500 milliseconds is an example value mentioned in specification section "4.10.1.5. Busy"
+    uint16_t minWaitTime    = 500; // milliseconds
```

Comment:
Yeah, this makes very little sense for CASE....  500 is just there to show how to encode the number into bytes.  The right thing for CASE is "how long will it take until the handshake is done", and that really depends.  A successful handshake will be several seconds.  One that times out might be 30s.

I would add a comment here explaining all that, set to 5s, file a followup to maybe come up with a better estimate.

Reply:
Added the comment, using 5 seconds, and filed #28288